libs/compare.go

```diff
@@ -0,0 +1,31 @@
+package libs
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+)
+
+func StreamsEqual(s1 io.Reader, s2 io.Reader) (bool, error) {
+	const chunkSize = 64000
```

Comment:
I think it's a little too much. We can use one of default sizes from `bufio.NewReader` or `io.Copy` (also a power of two). But I don't feel too strongly about it

Reply:
fixed, use bufio.NewReader def val